demonstrations/tutorial_multiclass_classification.py

```diff
@@ -90,14 +94,21 @@ def layer(W):
 # Data is embedded in each circuit using amplitude embedding:
 
 def circuit(weights, feat=None):
-    qml.templates.embeddings.AmplitudeEmbedding(feat, [0, 1], pad=0.0, normalize=True)
+    qml.templates.embeddings.AmplitudeEmbedding(feat, range(0,num_qubits), pad=0.0, normalize=True)
```

Comment:
Should do the same thing :)

```suggestion
    qml.templates.embeddings.AmplitudeEmbedding(feat, range(num_qubits), pad=0.0, normalize=True)
```

Reply:
:)
yes that's better.